src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -1,17 +1,144 @@
+  const classes = useStyles();
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    watch,
+    reset,
+    setValue,
+  } = useForm<any>();
+
+  // Setup watches
+  const empType = watch("empType");
+  const isNewCivMil = watch("isNewCivMil");
+  const hasExistingCAC = watch("hasExistingCAC");
+  const eta = watch("eta");
+
+  const gradeRankOptions: IComboBoxOption[] = useMemo(() => {
+    switch (empType) {
+      case EMPTYPES.Civilian:
+        return [...GS_GRADES, ...NH_GRADES];
+      case EMPTYPES.Military:
+        return [...MIL_GRADES];
+      case EMPTYPES.Contractor:
+        return [];
+      default:
+        return [];
+    }
+  }, [empType]);
+
+  const minCompletionDate: Date = useMemo(() => {
+    // Set the minimumn completion date to be 14 days from the estimated arrival
+    if (eta) {
+      let newMinDate = new Date(eta);
+      newMinDate.setDate(newMinDate.getDate() + 14);
+      return newMinDate;
+    } else return new Date();
+  }, [eta]);
+
+  const onOpen = () => {
+    const transRes = {
+      ...props.data,
+      hasExistingCAC: props.data?.hasExistingCAC ? "yes" : "no",
+      isNewCivMil: props.data?.isNewCivMil ? "yes" : "no",
+      isNewToBaseAndCenter: props.data?.isNewToBaseAndCenter ? "yes" : "no",
+    };
+    //Populate the React-Hook-Form with the transformed data
+    reset(transRes);
+  };
+
+  const updateRequest: SubmitHandler<IInRequest> = (data) => {
+    const requestApi = RequestApiConfig.getApi();
+    /* Validation has passed, so update the request */
+
+    // Transform "yes" / "no" to true/false
+    const hasExistingCAC = data?.hasExistingCAC ? true : false;
+    const isNewCivMil = data?.isNewCivMil ? true : false;
+    const isNewToBaseAndCenter = data?.isNewToBaseAndCenter ? true : false;
+
+    let dataCopy = {
+      ...data,
+      hasExistingCAC,
+      isNewCivMil,
+      isNewToBaseAndCenter,
+    };
+
+    // If it isn't a Civ/Mil, ensure values depending on Civ/Mil only are set correctly
+    if (
+      dataCopy.empType !== EMPTYPES.Civilian &&
+      dataCopy.empType !== EMPTYPES.Military
+    ) {
+      dataCopy.isNewCivMil = false;
+      dataCopy.prevOrg = "";
+      dataCopy.isNewToBaseAndCenter = false;
+    } else {
+      // If it is a new Civ/Mil then ensure prevOrg is set to ""
+      if (dataCopy.isNewCivMil === false) {
+        dataCopy.prevOrg = "";
+      }
+    }
+
+    if (dataCopy.empType !== EMPTYPES.Contractor) {
+      // If Employee is not a CTR then we should set hasExistingCAC to false and CACExpiration to undefined
+      dataCopy.hasExistingCAC = false;
+      dataCopy.CACExpiration = undefined;
+    } else {
+      if (!dataCopy.hasExistingCAC) {
+        // If the CTR doesn't have an Existing CAC, set the CACExpiration to undefined
+        dataCopy.CACExpiration = undefined;
+      }
+    }
+    requestApi.updateItem(dataCopy);
+    //setFormData(dataCopy);
+    props.hideEditPanel();
```

Comment:
Think this should be
```suggestion
    props.hideEditPanel();
```